src/ddl/dashboard_signal.sql

```diff
@@ -0,0 +1,22 @@
+/*
+
+This table stores the signals used in the public signal dashboard.
+
++---------+--------------+------+-----+---------+----------------+
+| Field   | Type         | Null | Key | Default | Extra          |
++---------+--------------+------+-----+---------+----------------+
+| id      | int(11)      | NO   | PRI | NULL    | auto_increment |
+| name    | varchar(255) | NO   |     | NULL    |                |
+| source  | varchar(32)  | YES  |     | NULL    |                |
+| enabled | binary(1)    | YES  |     | NULL    |                |
++---------+--------------+------+-----+---------+----------------+
+
+*/
+
+CREATE TABLE `dashboard_signal` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `source` varchar(32) NOT NULL,
+  `enabled` binary(1) NOT NULL,
+  PRIMARY KEY (`id`)
```

Comment:
To prevent us from accidentally creating duplicates? I feel like for now that won't matter (because we can track it ourselves) and we may want the flexibility of having dupes where one is disabled the other enabled if we want to migrate. I guess I don't feel too strongly about it though, especially since this table will likely stay small (<20 rows).

Reply:
(if we anticipate each source-signal pair being its own dashboard signal, that's 173 rows)

Reply:
I was thinking we would be more likely to have a single signal/source pair representing the entire source -- but I guess we don't actually know how it will end up.